packages/solid-tiptap/README.md

```diff
@@ -83,9 +73,13 @@ createEffect(() => {
 - `useEditorIsActive`: reactively subscribe to `isActive`.
 
 ```js
-const isHeading = useEditorIsActive(() => props.editor, () => 'heading', {
-  level: 1,
-});
+const isHeading = useEditorIsActive(
+  () => props.editor,
+  () => "heading",
+  {
+    level: 1,
+  }
+);
```

Comment:
Some changes in this PR were made by my formatter, let me know if it's good or I should change it.